elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -180,41 +199,46 @@ async def _verify_attached_flaws_for(self, advisory_id: int, attached_trackers:
+                self._complain(f"On advisory {advisory_id}, {cve} is associated with Brew components "
+                               f"{', '.join(sorted(cve_package_exclusions))} without a tracker bug."
+                               " You may need to explictly exclude those Brew components from the CVE "
+                               "mapping or attach the corresponding tracker bugs.")
 
         # Check if flaw bugs match the CVE field of the advisory
         advisory_cves = advisory_info["content"]["content"]["cve"].split()
         extra_cves = cve_components_mapping.keys() - advisory_cves
         if extra_cves:
-            self._complain(f"On advisory {advisory_id}, bugs for the following CVEs are already attached but they are not listed in advisory's `CVE Names` field: {', '.join(sorted(extra_cves))}")
+            self._complain(f"On advisory {advisory_id}, bugs for the following CVEs are already attached "
+                           f"but they are not listed in advisory's `CVE Names` field: {', '.join(sorted(extra_cves))}")
         missing_cves = advisory_cves - cve_components_mapping.keys()
         if missing_cves:
-            self._complain(f"On advisory {advisory_id}, bugs for the following CVEs are not attached but listed in advisory's `CVE Names` field: {', '.join(sorted(missing_cves))}")
+            self._complain(f"On advisory {advisory_id}, bugs for the following CVEs are not attached but listed in "
+                           f"advisory's `CVE Names` field: {', '.join(sorted(missing_cves))}")
 
-    async def get_attached_bugs(self, advisory_ids: Iterable[str]) -> Dict[int, Set[Bug]]:
+    def get_attached_bugs(self, advisory_ids: Iterable[str]) -> (Dict[int, Set[Bug]], Dict[int, Set[Bug]]):
         """ Get bugs attached to specified advisories
-        :return: a dict with advisory id as key and set of bug objects as value
+        :return: 2 dicts (one for jira bugs, one for bz bugs) with advisory id as key and set of bug objects as value
         """
-        green_print(f"Retrieving bugs for advisory {advisory_ids}")
-        if self.use_jira:
-            issue_keys = {advisory_id: [issue["key"] for issue in errata.get_jira_issue_from_advisory(advisory_id)] for advisory_id in advisory_ids}
-            bug_map = self.bug_tracker.get_bugs_map([key for keys in issue_keys.values() for key in keys])
-            result = {advisory_id: {bug_map[key] for key in issue_keys[advisory_id]} for advisory_id in advisory_ids}
-        else:
-            advisories = await asyncio.gather(*[self.errata_api.get_advisory(advisory_id) for advisory_id in advisory_ids])
-            bug_map = self.bug_tracker.get_bugs_map(list({b["bug"]["id"] for ad in advisories for b in ad["bugs"]["bugs"]}))
-            result = {ad["content"]["content"]["errata_id"]: {bug_map[b["bug"]["id"]] for b in ad["bugs"]["bugs"]} for ad
-                      in advisories}
-        return result
```

Comment:
@joepvd Earlier in this method we were doing a combination of async calls and sync calls depending on jira/bz. As you can see both branches went into the nitty gritty, doing individual ET call for jira issue / parsing bug ids from advisory. We now have both of those things supported by errata_tool library, having them use the same methods that we use everywhere else in the codebase is a win for maintainability - and also testing. Since right now to support this I would add json parsing of jira issues - which errata_tool object invocation already does that for us (I added that support :)). The only "regression" if you want to call it that, is it's going to be slow - stuff is not going to be async, so result will be slower by a few seconds - I think that's a fair tradeoff for now. I definitely want to look into how we can make errata_tool calls async - in a different PR though